c/algebra.h

```diff
@@ -134,6 +136,144 @@ MATRIX* mult(MATRIX *A, MATRIX *B)
     return C;
 }
 
+MATRIX* id(uint32_t size)
+{
+    MATRIX *A = push_matrix(size, size);
+    if (A == NULL)
+    {
+        LOG_ERROR("Identity matrix was not created.");
+        return NULL;
+    }
+
+    for (uint32_t i = 0U; i < A->rows; i++)
+    {
+        A->val[A->cols * i + i] = 1.0F;
```

Comment:
it might be that writing a macro as `#define ENTRY_IN(mat, row, col) #mat->val[mat->cols * row + col]`, such that readability is improved